### DISCUSSION TITLE
Stepper Motors Cooler?

### DISCUSSION THREAD

**Discussion by Cal-LF** [Jyers/Marlin#130] [Q&A] [answered]
Using my own compiled FW (added filament change) of the V427 3 X 3 MM, the temperature of all the stepper motors are noticeably cooler. Is this a result of adjustments made in the FW?

**Comment by Schnabulation** [comment, marked as answer]
As far as I know the TMC2208 stepper drivers of the 4.2.2 and the 4.2.7 board are in legacy mode, so the voltage is **not** configurable through the firmware, only manually by turning the knob directly on the mainboard.

So it would really astonish me if it has anything to do with the firmware.

**Reply by Cal-LF** [reply]
Thanks for the info. I did set the vref when the board was installed and recorded the temps on the steppers after a several hour print. Can't say why they are running cooler now.
